user-profiles.md

```diff
@@ -13,21 +13,32 @@ When considering requirements and implementation solutions it's useful look at w
 
 Profiles describe a type of role a user may perform. A real person may perform more than one role and have more than one profile apply to them. How this mapping works between profiles and real people can vary between companies and other organizations. To handle this variation we focus on the user profiles rather than how they may map to people in these different organizations.
 
-### 1. Pipeline Operator
+### 1. Pipeline and Task Authors
 
-Pipeline operators define, schedule, monitor, and otherwise operate pipelines across one or more Kubernetes clusters. For example, the operation of a set of Kaniko build pipelines and Argo CD pipelines. This is not to be confused with the role of a Kubernetes cluster operator. This is also not to be confused with the application's operator role, who would oversee a product in production.
+Pipeline and Task authors are people who write Pipelines and Tasks. They may be also using them, or they may be creating Tasks and Pipelines for use by others.
 
-### 2. Pipeline Distributor
+For example:
+* Folks who maintain entries in [Tekton's Catalog](https://github.com/tektoncd/catalog) for Kaniko and ArgoCD.
+* Engineers on a platform team in a company who want to create standard Pipelines for use by multiple teams
 
-Distributors are people who package application pipelines for someone else to operate. Examples of this would be those who maintain [Tekton's Catalog](https://github.com/tektoncd/catalog) for Kaniko and ArgoCD.
+### 2. Pipeline and Task Users
+
+Pipeline and Task users are folks who find themselves actually running Pipelines and Tasks by creating or invoking tools that create, PipelineRuns and TaskRuns.
+
+For example:
+* An application developer who is focused on releasing product updates with high quality as quickly as possible may be using Tasks from [the Catalog](https://github.com/tektoncd/catalog) or provided by their company's platform team.
 
 ### 3. Platform Builder
 
-Platform builders are people who wrap Tekton's core engine and libraries to extend it. An example of this is the platform being built by [Jenkins X](https://github.com/jenkins-x/jx) using Tekton pipelines.
+Platform builders are people who wrap Tekton's core engine and libraries to extend it.
```

Comment:
Does platform builder also encompass someone who is writing a system which conforms to Tekton's spec but doesn't necessarily utilize Tekton's engine or libraries or doesn't run on Kubernetes?  Or is that maybe a different thing?   "API implementer" /  ... something else ... ?

Reply:
oooo that's a good question - maybe for now we can lump those folks in with other platform builder but if we have people coming along who want to do this, maybe we'll need to split it out 🤔

Reply:
I'm gonna try a "3a"